tests/lib/rules/no-sequences.js

```diff
@@ -15,10 +15,20 @@ var rule = require("../../../lib/rules/no-sequences"),
 // Tests
 //------------------------------------------------------------------------------
 
-var errors = [{
-    message: "Unexpected use of comma operator.",
-    type: "SequenceExpression"
-}];
+/**
+ * Create error message object for failure cases
+ * @param {int} column column of the error
+ * @returns {object} returns the error messages collection
+ * @private
+ */
+function errors(column) {
+    return [{
+        message: "Unexpected use of comma operator.",
+        type: "SequenceExpression",
+        line: 1,
```

Comment:
I don't think this is right. This will always report an error on the first line. Your unit tests are passing, because you only have 1 line in all of them.